index.html

```diff
@@ -135,9 +151,10 @@ <h3 class="year">2015</h3>
         
         </div>
     </article>
-    <article>
+    <article class="cards">
         <div class="container-2">
-            <a href="#" class="project-pics"><img class="project-pics" src="./images/Snapshoot Portfolio (2).png" alt=""></a>
+            <a href="#" class="mobile-pic" ><img class="project-pics" src="./images/Snapshoot Portfolio (3).png" alt=""></a>
+            <a href="#" class="desktop-pic" ><img class="project-pics" src="./images/3d.png" alt=""></a>
```

Comment:
:red_circle: Your project card images for the desktop version are not displaying. Please make sure the images are present and the paths are correct 

Your Design
![noimage](https://user-images.githubusercontent.com/26736582/199906219-feaf0b0e-e5cd-41df-a9eb-b299cc36fd1b.png)